api_tests/requests/views/test_node_request_list.py

```diff
@@ -0,0 +1,273 @@
+from unittest import mock
```

Comment:
This test file was getting long and convoluted, so I split out the preprint tests.